src/layers/src/geojson-layer/geojson-utils.ts

```diff
@@ -138,6 +141,17 @@ export function parseGeometryFromString(geoString: string): Feature | null {
     }
   }
 
+  // try parse as wkb using loaders.gl WKBLoader
+  if (!parsedGeo) {
+    try {
+      const buffer = Buffer.from(geoString, 'hex');
```

Comment:
Nit: Wish we had a loader for this so we didn't need to mess with Node.js Buffer class. I will give it some thought.

Reply:
Thanks! Ib will address this in https://github.com/visgl/loaders.gl/pull/2652